Mu2eKinKal/test/KKLineSeed.fcl

```diff
@@ -83,13 +85,10 @@ outputs : {
 }
 
 #include "Production/JobConfig/reco/epilog.fcl"
-#include "Offline/CRVResponse/fcl/epilog_extracted_v02.fcl"
+#include "Offline/CRVResponse/fcl/epilog_extracted_v03.fcl"
```

Comment:
There is now a generic epilog_extracted.fcl that references the current version, please use that instead